crates/astria-sequencer/src/grpc/storage/keys.rs

```diff
@@ -0,0 +1,71 @@
+use astria_core::primitive::v1::RollupId;
+use base64::{
+    display::Base64Display,
+    engine::general_purpose::URL_SAFE,
+};
+
+pub(in crate::grpc) fn block_hash_by_height(height: u64) -> String {
+    format!("grpc/block_hash/{height}")
+}
+
+pub(in crate::grpc) fn sequencer_block_header_by_hash(hash: &[u8; 32]) -> String {
+    format!("grpc/block_header/{}", Base64Display::new(hash, &URL_SAFE))
+}
+
+pub(in crate::grpc) fn rollup_data_by_hash_and_rollup_id(
+    hash: &[u8; 32],
+    rollup_id: &RollupId,
+) -> String {
+    format!(
+        "grpc/rollup_data/{}/{}",
+        Base64Display::new(hash, &URL_SAFE),
+        Base64Display::new(rollup_id.as_ref(), &URL_SAFE)
```

Comment:
We should make the rollup ID display impl URL safe in general.

We have a chance to still change the formatting of rollup IDs - should we just do this in this PR?

Reply:
Done in [4db1136](https://github.com/astriaorg/astria/pull/1616/commits/4db1136280c76390790d920d2f3db9dec6b3ab10).